contribs/gmf/less/desktop.less

```diff
@@ -101,11 +101,15 @@ main {
       background-color: fade(@main-bg-color, 80%);
       border-bottom-left-radius: 0;
       border-bottom-right-radius: 0;
-      right: @app-margin;
+      padding: 0;
+      left: 50%;
+      @width: 4rem;
```

Comment:
I just want to make sure that `margin-left` is half `width`. This is why I'm using a variable.